base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,59 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+    WCHAR szPath[MAX_PATH];
+    LPWSTR szExplorer;
+    BOOL bIsSystemShell = TRUE; // Assume we are the system shell by default.
+
+    if (!GetModuleFileNameW(NULL, szPath, _countof(szPath)))
+        return FALSE;
+
+    szExplorer = PathFindFileNameW(szPath);
+
+    if (RegOpenKeyExW(HKEY_LOCAL_MACHINE, L"Software\\Microsoft\\Windows NT\\CurrentVersion\\Winlogon",
+        0, KEY_READ, &hKeyWinlogon) != ERROR_SUCCESS)
+    {
+        // No registry access.
+        bIsSystemShell = TRUE;
+    }
+    else
+    {
+        DWORD cbShell;
+        LSTATUS status;
```

Comment:
```suggestion
        LSTATUS Status;
```
And update elsewhere. Lower case is for Hungarian prefixes.

From our coding style:
> Capitalize names of variables and functions.
[Hungarian Notation](https://reactos.org/wiki/Hungarian_Notation) may be used when developing for Win32, but it is not required. If you don’t use it, the first letter of a name must be a capital too (no lowerCamelCase). Do not use underscores as separators either.